interface/batchcom/batchcom.php

```diff
@@ -24,280 +29,217 @@
-        <?php $datetimepicker_showseconds = false; ?>
-        <?php $datetimepicker_formatInput = false; ?>
-        <?php require($GLOBALS['srcdir'] . '/js/xl/jquery-datetimepicker-2-5-4.js.php'); ?>
-        <?php // can add any additional javascript settings to datetimepicker here; need to prepend first setting with a comma ?>
-    });
-});
-</script>
-
+<?php Header::setupHeader(['datetime-picker']); ?>
+<style>
+.datepicker {
+    width: 100%;
+}
+</style>
 </head>
+
 <body class="body_top">
 <!-- larry's sms/email notification -->
-<span class="title"><?php include_once("batch_navigation.php");?></span>
+<?php include_once("batch_navigation.php");?>
 <!--- end of larry's insert -->
-<span class="title"><?php xl('Batch Communication Tool','e')?></span>
-<br><br>
-
-<!-- for the popup date selector -->
-<div id="overDiv" style="position:absolute; visibility:hidden; z-index:1000;"></div>
-<FORM name="select_form" METHOD=POST ACTION="" onsubmit='return top.restoreSession()'>
-<div class="text">
-<div class="main_box">
-    <table class="table" ><tr><td >
-        <?php
-        if ($form_err) {
-            echo (xl('The following errors occurred')."<br>$form_err<br><br>");
+<main class="container">
+    <header class="row">
+        <div class="col-md-6 col-md-offset-3 text-center">
+            <h1><?php xl('Batch Communication Tool','e')?></h1>
+        </div>    
+    </header>
+    <?php if ($form_err) { echo "<div class=\"alert alert-danger\">".xl("The following errors occurred").": $form_err</div>"; } ?>
+    <form name="select_form" method="post" action="">
+        <div class="row">
+            <div class="col-md-3 well">
+                <label for="process_type"><?php echo xl("Process","e").":"; ?></label>
+                <select name="process_type">
+                    <?php foreach ($process_choices as $choice) { echo "<option>$choice</option>"; } ?>
+                </select>
+            </div>
+            <div class="col-md-3 well">
+                <label for="hipaa_choice"><?php echo xl("Override HIPAA choice").":"; ?></label>
+                <select name="hipaa_choice">
+                    <?php foreach ($hipaa_choices as $choice) { echo "<option>$choice</option>"; } ?>
+                </select>
+            </div>
+            <div class="col-md-3 well">
+                <label for="sort_by"><?php echo xl("Sort by","e"); ?></label>
+                <select name="sort_by">
+                    <?php foreach ($sort_by_choices as $choice => $sorting_code) { echo "<option value=\"$sorting_code\">$choice</option>"; } ?>
+                </select>
+            </div>
+            <div class="col-md-3 well">
+                <label for="age_from"><?php echo xl("Age Range","e").":"; ?></label>
+                <input name="age_from" size="2" type="num" placeholder="<?php echo xl("any"); ?>"> - <input name="age_upto" size="2" type="num" placeholder="<?php echo xl("any"); ?>">
```

Comment:
and here's a good example of xla use:
``````
<input name="age_from" size="2" type="num" placeholder="<?php echo xla("any"); ?>"> - <input name="age_upto" size="2" type="num" placeholder="<?php echo xla("any"); ?>">